Cyborg/VectorView.swift

```diff
@@ -90,9 +90,70 @@ open class VectorView: UIView {
 
     open override func layoutSubviews() {
         super.layoutSubviews()
-        if bounds.size != .zero {
+        func makeActualSize(_ drawable: VectorDrawable,
```

Comment:
Personally not a fan of hidden anonymous functions like this because I would typically think you would want to unit test them but that's almost always be being pedantic.

Reply:
This should be tested with a snapshot test imo. But we don't have them yet in this library.

Reply:
Yeah I was gonna suggest that but when I saw they weren't setup I figured it wasn't worth calling out. I figured snapshot tests were a goal at some point.